Helper/Api/Products.php

```diff
@@ -1024,7 +1029,7 @@ private function processProduct(
             $target->setWebsiteIds($websiteIds);
         }
 
-        $shouldUpdate = $this->processImages($flat_product, $target, $shouldUpdate);
+        $target = $this->processImages($flat_product, $target, $shouldUpdate);
```

Comment:
where was the part that we do not update the image if there is are gif images present?